java/hadoop/src/main/java/com/youtube/vitess/hadoop/VitessConf.java

```diff
@@ -97,4 +99,12 @@ public void setAlgorithm(SplitQueryRequest.Algorithm algorithm) {
   public void setRpcFactoryClass(Class<? extends RpcClientFactory> clz) {
     conf.set(RPC_FACTORY_CLASS, clz.getName());
   }
+
+  public Query.ExecuteOptions.IncludedFields getIncludedFields() {
+    return conf.getEnum(INCLUDED_FIELDS, Query.ExecuteOptions.IncludedFields.TYPE_AND_NAME);
+  }
+
+  public void setIncludedFields(Query.ExecuteOptions.IncludedFields includedFields) {
```

Comment:
It will work as is, but any hadoop jobs launched in the meantime will use the default value for Includedfields. They can still call this setter directly, if they want.